esphome/components/rtsp_server/__init__.py

```diff
@@ -0,0 +1,36 @@
+import esphome.config_validation as cv
+import esphome.codegen as cg
+from esphome.const import (
+    CONF_ID,
+    CONF_PORT
+);
+from esphome.core import coroutine_with_priority, CORE
+from esphome.components import esp32_camera
+
+
+CODEOWNERS = ["@crossan007"]
+DEPENDENCIES = ["network"]
```

Comment:
```suggestion
DEPENDENCIES = ["network", "esp32_camera"]
```

Now `esp32_camera` _should_ be a dependency, as `rstp_server` relies directly on it (see the `#include`s)